prometheus/counter.go

```diff
@@ -88,8 +89,9 @@ type counter struct {
 	// valInt stores values that are exact integers. Both have to go first
 	// in the struct to guarantee alignment for atomic operations.
 	// http://golang.org/pkg/sync/atomic/#pkg-note-BUG
-	valBits uint64
-	valInt  uint64
+	valBits     uint64
+	valInt      uint64
+	createdTime uint64
```

Comment:
I don't think this field is ever used, no?

Reply:
It's initialized on L166 in `initCreated()` call.

Presumably, a forthcoming change would use it for serving in a Prometheus counter metric.

Reply:
Yup, but we would need this to be either in this PR ideally so we see how end solution would look like 🤗 

Definitely it's good direction!